boilerplate/Dockerfile

```
@@ -0,0 +1,18 @@
+FROM openjdk:17-jdk-slim
+
+WORKDIR /app
+
+COPY build.gradle settings.gradle ./
+COPY gradle ./gradle
+COPY gradlew ./
+COPY src ./src
+
+RUN chmod +x gradlew
+
+RUN ./gradlew build --no-daemon -x test
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)